src/main/java/pages/HomePageLuma.java

```diff
@@ -0,0 +1,34 @@
+package pages;
+import org.openqa.selenium.WebDriver;
+
+public class HomePageLuma extends BasePage {
+
+
+    public HomePageLuma(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
```

Comment:
all the methods that are from here to the bottom, are not necessary since the methods are already implemented in `BasePage()` and are being inherited, therefore delete all the elements below